src/Core/ConstraintValidator/Constraints/TypedRegex.php

```diff
@@ -52,6 +52,7 @@ class TypedRegex extends Constraint
     public const TYPE_DNI_LITE = 'dni_lite';
     public const TYPE_UPC = 'upc';
     public const TYPE_EAN_13 = 'ean_13';
```

Comment:
It seems to me that the constant is no longer used internally. Maybe depreciated it?